app/bundles/CampaignBundle/Tests/CampaignTestAbstract.php

```diff
@@ -80,7 +82,15 @@ protected function initCampaignModel()
             ->method('getRepository')
             ->will($this->returnValue($formRepository));
 
-        $campaignModel = new CampaignModel($coreParametersHelper, $leadModel, $leadListModel, $formModel);
+        $eventCollector = $this->getMockBuilder(EventCollector::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $removedContactTracker = $this->getMockBuilder(RemovedContactTracker::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
Can be one liner: `$removedContactTracker = $this->createMock(RemovedContactTracker::class);`